kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -198,19 +195,9 @@
         activeResourceMap,
         replacementQuestionPool,
         clearSelectedQuestions,
-        replaceSelectedQuestions,
-        toggleQuestionInSelection,
         handleReplacement,
-        allSections,
       } = injectQuizCreation();
 
-      const activeSectionTitle = computed(() => {
-        const activeSectionIndex = allSections.value.findIndex(section =>
-          isEqual(JSON.stringify(section), JSON.stringify(activeSection.value)),
-        );
-        return displaySectionTitle(activeSection.value, activeSectionIndex);
-      });
-
```

Comment:
@nucleogenesis there's a fair amount of stuff here that's unused in the create exam page, and I think (?) you might be best positioned to take a look and confirm what of this was just leftover from refactoring and other adjustments at the end of 0.17. It seems _mostly_ like that, but... could you just take an extra look through this? I think you will have a better sense than I do.